test/support/exception_notifier_helper.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# this extension allows ExceptionNotifier to reset all the glocal settings
+# (i.e. class vars that otherwise remains during the test)
+# please remembeer to call this method each time after you set such settings
+# to prevent order dependent test fails.
+module ExceptionNotifier
```

Comment:
As I described in the code, we would encounter random test fail unless we reset ExceptionNotifier's global settings properly.
When we play around the settings in the test codes, the class variables-based settings remain after the test case, which sometimes screws up the next cases that relies on them.

This helper is intended to fix the misbehavior by making sure to reset those global settings at teardown phase.